tests/spec/conftest.py

```diff
@@ -58,7 +58,7 @@ def run_test(table_data, series, expected_results):
         # variable which is the series under test.
         dataset = Dataset()
         # TODO: See temporary hack above
```

Comment:
Is this still TODO? And is the hack still needed? You closed https://github.com/opensafely-core/databuilder/issues/438 the other day.

Reply:
I was going to address that in a separate PR. I think some of the spec examples might need changing as they embed incorrect population semantics.

Reply:
I think really this comment is a misplaced raised eyebrow in the direction of you closing #438 when I don't think it's really resolved.

Reply:
I think that is accurate!